File: modulo1/segmento3/apunte3/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Ingrese un numero del 1 al 7 y se devolvera un dia de semana: ");
		Integer num = Integer.parseInt(keyboard.nextLine());

		if(num == 1) {
			System.out.println("Lunes");
		}else if(num == 2){
			System.out.println("Martes");
		}else if(num == 3){
			System.out.println("Miercoles");
		}else if(num == 4){
			System.out.println("Jueves");
		}else if(num == 5){
			System.out.println("Viernes");
		}else if(num == 6){
			System.out.println("Sabado");
		}else if(num == 7){
			System.out.println("Domingo");
		}
		else {
			System.out.println("Opcion Invalida");
		}
	}
}

File: modulo1/segmento3/apunte3/ejercicio2.js
function ejercicio2(){
	const num = parseInt(prompt('Ingrese un  umero del 1 al 7 y se devolvera un dia de semana: '));

	if(num == 1){
		alert('Lunes');
	} else if (num == 2){
		alert('Martes');
	} else if (num == 3){
		alert('Miercoles');
	} else if (num == 4){
		alert('Jueves');
	} else if (num == 5){
		alert('Viernes');
	} else if (num == 6){
		alert('Sabado');
	} else if (num == 7){
		alert('Domingo');
	} else {
		alert('Opcion invalida');
	}
}

